cf-agent/package_module.c

```diff
@@ -676,7 +676,7 @@ int UpdatePackagesDB(Rlist *data, const char *pm_name, UpdateType type)
                                              package_data[1], package_data[2],
                                              type);
 
-                        BufferAppendF(inventory_data, "%s,%s,%s\n",
+                        BufferAppendF(inventory_data, "\"%s\",\"%s\",\"%s\"\n",
```

Comment:
This could be problematic if one of the fields contain quotes. Maybe we should use the `csv_writer.h` functions?